storage/clean/oldDatabaseCleaner.go

```diff
@@ -192,7 +192,9 @@ func (odc *oldDatabaseCleaner) cleanOldEpochs(currentEpoch uint32) error {
 		return nil
 	}
 
+	wasCleaned := false
```

Comment:
You can replace this line with:
```
if len(sortedEpochs) == 0 {
    return nil
}
```
and remove L197, L210 and L213

Reply:
as discussed, will remove this, since it's ok to clean the map for the case when db deletion is set to false, then the map won't grow continuously